src/app/space-settings/areas/create-area-dialog/create-area-dialog.component.html

```diff
@@ -7,6 +7,12 @@ <h3 class="create-dialog-header">
   <section>
     <div class="form">
       <fieldset class="create-fieldset">
+        <div *ngIf="errors" class="alert alert-danger">
+          <div [hidden]="!errors.unique">
+            <span class="pficon pficon-error-circle-o"></span>
+            <strong>Unique Name</strong> A unique name is required to add a new area.
```

Comment:
change to `<b>Unique Name</b>`, as `<strong>` is a logical state and this only needs to be a physical state of the text. i.e. only bolded.